src/__tests__/fixtures/color-case.compressed.css

```diff
@@ -1 +1 @@
-p{color:#AA93FE;background-color:#AFE}
\ No newline at end of file
+p{color:#AA93FE;background:#FFF url(/ImagesDir/loader.gif) center center no-repeat}
```

Comment:
If we are applying this transform to the other formats then it makes sense to do it here also, right?